src/main/java/banking/Menu.java

```diff
@@ -0,0 +1,168 @@
+        scanner = new Scanner(System.in);
+    }
+
+    public void showMenu() {
+        int choose;
+
+        while (true) {
+            if (loggedIn) {
+                loggedMenu();
+            } else {
+                unloggedMenu();
+            }
+
+            choose = scanner.nextInt();
+            if (loggedIn) {
+                loggedInControl(choose);
+            } else {
+                loggedOutControl(choose);
+            }
+        }
+    }
+
+    private void loggedOutControl(int choose) {
+        switch (choose) {
+            case 1:
+                createAccountOption();
+                break;
+            case 2:
+                logInOption();
+                break;
+            case 0:
+                System.out.println("Bye!");
+                exit(0);
+            default:
+                System.out.println("Wrong number");
+                break;
+        }
+    }
+
+    private void loggedInControl(int choose) {
+        switch (choose) {
+            case 1:
+                System.out.println("Balance: " + operations.getBalance(id));
+                break;
+            case 2:
+                System.out.println("Enter income:");
+                int amount = scanner.nextInt();
+                operations.addBalance(id, amount);
+                break;
+            case 3:
+                transferToOption();
+                break;
+            case 4:
+                operations.delAccount(id);
+                System.out.println("The account has been closed!");
+                loggedIn = false;
+                id = -1;
+                break;
+            case 5:
+                break;
+            case 0:
+                System.out.println("Bye!");
+                exit(0);
+            default:
+                System.out.println("Wrong number");
+                break;
+        }
+    }
+
+    private void transferToOption() {
+        System.out.println("Enter card number:");
+        String card = scanner.next();
+
+        if (!card.equals(operations.getNumber(id))) {
+            if (cardValidator.isValidCard(card)) {
+
+                if (operations.containCard(card)) {
+                    System.out.println("Enter how much money you want to transfer:");
+                    int amountSend = scanner.nextInt();
+                    if (amountSend < operations.getBalance(id)) {
+                        operations.sendMoney(id, card, amountSend);
+                        System.out.println("success");
+                    } else {
+                        System.out.println("Not enough money!");
+                    }
+                } else {
+                    System.out.println("Such a card does not exist.");
+                }
+            } else {
+                System.out.println("Probably you made a mistake in the card number. Please try again!");
+            }
+        } else {
+            System.out.println("You can't transfer money to the same account!");
+        }
```

Comment:
Can you keep each `if-block` for validation separately without a nested chain of `if-else`?

Reply:
fixed